modules/c++/re/include/re/PCRE.h

```diff
@@ -171,10 +187,22 @@ class PCRE
 
 protected:
     std::string mPattern;
+
+#ifdef __CODA_CPP11
+    /*!
+     *  Replace non-escaped "." with "[\s\S]" to get PCRE_DOTALL newline behavior
+     *  \param str  The string to modify
+     *  \return  The modified string
+     */
+    std::string replace_dot(const std::string& str) const;
+
+    std::regex mRegex;
+#else
     //! The pcre object
     pcre *      mPCRE;
     //! The output/offset vector
     int         mOvector[OVECCOUNT];
```

Comment:
I know this predates your changes but any reason why the `OVECCOUNT` constant that's at the top of this file currently can't move to be a private constant?